mb_modelbase/cache/BaseCache.py

```diff
@@ -0,0 +1,80 @@
+import abc
+import dill
+from typing import List
+
+
+def model_key(name: str, model: List[str], where: List[str]) -> str:
+    """Compute a key for a given model"""
+    return (str(name) + ':' + str(model).strip('[]') + ':' + str(where).strip('[]')).replace(' ', '')
+
+
+def predict_key(base, predict_stmnt: List[str], where_stmnt, splitby_stmnt) -> str:
+    """Compute a key for a given prediction query"""
+    return str(base) + str(predict_stmnt) + str(where_stmnt) + str(splitby_stmnt)
+
+
+class BaseCache(abc.ABC):
+    """
+    Abstract baseclass for modelbase caches
+
+    Override function _get and _set to use various storage backends
+
+    Attributes:
+        _serialize (bool): sets if payload gets serialized before storage
+
+    """
```

Comment:
Adhere to PEP 257 (https://www.python.org/dev/peps/pep-0257/).
In particular but not only: (1) leading """ is not on a separate line. (2) Trailing """ is not not preceeded by an entirely empty line, and (3) docstrings should be complete sentences.

Reply:
Changing (1), even though leading """ on a seperate line is in the allowed by PEP 257.
Fixed (2) and (3)